src/main/java/net/minecraftforge/fml/client/FMLClientHandler.java

```diff
@@ -265,16 +265,21 @@ public void beginMinecraftLoading(Minecraft minecraft, List<IResourcePack> resou
         {
             Loader.instance().preinitializeMods();
         }
-        catch (CustomModLoadingErrorDisplayException custom)
-        {
-            FMLLog.log.error("A custom exception was thrown by a mod, the game will now halt", custom);
-            customError = custom;
-        }
         catch (LoaderException le)
```

Comment:
This exception is client side, the loader is common so it needs to be dealt with in the FMLClientHandler.